src/main/java/com/orbitz/consul/cache/ConsulCache.java

```diff
@@ -202,6 +202,14 @@ public boolean addListener(Listener<K, V> listener) {
         return added;
     }
 
+    public Iterable<Listener<K, V>> getListeners() {
```

Comment:
I'd suggest returning an ImmutableList or Collections.unmodifiable list here, it's generally iffy to give control of our member variables to other classes. 

For example, if in the future additional logic is add in to addListener/removeListener then this could give people a way to remove listeners without triggering that logic.